### ISSUE TITLE
Combine helper attributes into a single attribute

### ISSUE THREAD

**Issue by calebzulawski** [calebzulawski/multiversion#27]
Helper attributes are generally fine, but they could conflict with an actual macro (for example an attribute macro named `clone` wouldn't be unusual).

They could be combined into a single attribute instead:

```
#[multiversion(
    clone(...),
    clone(...),
)]
```

**Comment by est31** [comment]
:+1: it would also make conditional use of the multiversion crate easier.

**Comment by calebzulawski** [comment]
I didn't even consider that, but that's a good point!

**Comment by est31** [comment]
For reference, right now I have to do sth like:
```Rust
#[cfg_attr(feature="multiversion", multiversion)]
#[cfg_attr(feature="multiversion", clone(target = "[x86|x86_64]+avx+avx2+sse+sse2+sse3"))]
#[cfg_attr(feature="multiversion", clone(target = "[arm|aarch64]+neon"))]
fn foo() {}
```

It would be much nicer if I could do:

```Rust
#[cfg_attr(feature="multiversion", multiversion(
    clone(target = "[x86|x86_64]+avx+avx2+sse+sse2+sse3"),
    clone(target = "[arm|aarch64]+neon"),
))]
fn foo() {}
```

**Comment by calebzulawski** [comment]
Implemented in 16c3aa1baf1a56c743e7e9b65a2cfcad0cf4feb9.
